frontend/src/components/SearchTable.vue

```diff
@@ -93,35 +93,33 @@
                 <b>
                   <i>{{ searchedTerm }}</i>
                 </b>
-                <br />
-                If this is an alias or external identifier, it means it is not present in any of the
-                models.
+                among all {{ Object.keys(models).length }} integrated GEMs
               </div>
               <div class="content">
-                <span>Metabolites</span>
+                <h5 class="title is-5">You may search for the following items:</h5>
+                <h6 class="title is-6 m-0">Metabolites</h6>
                 <ul>
-                  <li>ID</li>
-                  <li>Name or aliases</li>
-                  <li>Formula without charge</li>
-                  <li>External identifiers</li>
+                  <li>ID (e.g. MAM03146l)</li>
+                  <li>Name or aliases (e.g. VLDL remnant)</li>
+                  <li>Formula without charge (e.g. C136286H232084N13476O38613P3271S174R7267)</li>
```

Comment:
Haha `C136286H232084N13476O38613P3271S174R7267` is obv. correct but it is very long, maybe something shorter like `C8H10NO2` of `MAM01738p` would make sense?

Reply:
changed in 0faf5103

Reply:
It seems to be the same long formula still though?

Reply:
Sorry, I forgot to push the changes